js/utils/mutation_handlers/cached_handler.js

```diff
@@ -15,13 +15,11 @@ let isNotCached = timeStamp => title => {
     return true;
   }
   return false;
-}
+};
 
 /**
  * Calls the callback if an event in the mutation list has a different to title
  * to or it occurs 10ms after it's precursor.
  */
 export const handler = callback => mutationList =>
-  (mutationList.some(x => isNotCached(Date.now())(x.target.innerHTML))) ? callback() : undefined;
-  
-
+  mutationList.some(x => isNotCached(Date.now(), x.target.innerHTML)) ? callback() : undefined;
```

Comment:
tbh this should probably be an `if` instead of a ternary, as callback isn't pure, and nothing is using the result

Reply:
I'll change it to an if.